python/lsst/analysis/tools/tasks/diaSourceTableVisitAnalysis.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from lsst.pipe.base import connectionTypes as ct
+
+from .base import AnalysisBaseConfig, AnalysisBaseConnections, AnalysisPipelineTask
+
+
+class DiaSourceTableCcdVisitAnalysisConnections(
+    AnalysisBaseConnections,
+    dimensions=("visit", "band"),
+    defaultTemplates={"coaddName": "deep", "fakesType": "fakes_"},
```

Comment:
We're trying to move to SI rather than fakes, siType and si_.

Reply:
We use `fakesType` [throughout](https://github.com/search?q=org%3Alsst+fakesType&type=code) AP, including in the upstream data products, so I propose to maintain it here.

Reply:
Benching this discussion for now to be readdressed later.